certified-connectors/DocuMotor/Readme.md

```diff
@@ -0,0 +1,40 @@
+
+## DocuMotor Connector
```

Comment:
@sebastianehlers please include all required section in the readme file. Please follow our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md).

Reply:
Aligned with supplied template.